pipdeptree.py

```diff
@@ -755,6 +768,11 @@ def get_parser():
                             'Comma separated list of select packages to exclude '
                             'from the output. If set, --all will be ignored.'
                         ), metavar='PACKAGES')
+    parser.add_argument('--exclude-regex', dest='exclude_regex',
```

Comment:
No `--include-regex' similar to `--exclude-regex'?

Reply:
TODO